main.go

```diff
@@ -87,8 +88,7 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
Just dumb question: Is it removed just because it is supposed to be deprecated? I was wondering if it is okay for removal. Thanks.

Reply:
Correct, `Port` deprecated (and moved to `manager.WebhookServer`) but it's also a Webhook specific setting we don't use so can be dropped.